src/client/components/app/Transactions/DepositTx.tsx

```diff
@@ -54,6 +55,9 @@ export const DepositTx: FC<DepositTxProps> = ({
   const [debouncedAmount, isDebouncePending] = useDebounce(amount, 500);
   const [txCompleted, setTxCompleted] = useState(false);
   const [isFetchingAllowance, setIsFetchingAllowance] = useState(false);
+  const servicesEnabled = useAppSelector(AppSelectors.selectServicesEnabled);
+  const simulationsEnabled = servicesEnabled.tenderly;
```

Comment:
nit: Wondering if we should be more explicit on which service we are using so that we know that it's tenderly that is down looking at the code
```suggestion
  const tenderlyEnabled = servicesEnabled.tenderly;
```

Reply:
did thought about it, but also thought we should have a fallback service in the future like

```
const simulationsEnabled = servicesEnabled.tenderly || servicesEnabled.alternateService;
```

And only turn off simulations when both are down...

Reply:
In that case, I'd try and centralize that knowledge and have an ExternalServiceStatus service that understands the domain of external services, and then we would have a function on there `isSimulationEnabled()` that would check `tenderly` and then the `alternateService`. It feels a bit too much for the `DepositTx` component to have such knowledge IMO

Reply:
yeah, you are right. I actually though of adding that since we will need it. There should be a common selector that aggregates all those, specially since there will also be another state for user settings, where user can disable external services manually if they wish, for privacy reasons. It just felt out of scope right now.

Think my example was wrong then, and meant to be something like:
```ts
const simulationsEnabled = useAppSelector(selectIsSimulationEnabled);
```
And that is why I chose that specific naming

Reply:
Sounds good, let's leave it for now